environment.yml

```diff
@@ -173,4 +173,5 @@ dependencies:
       - matplotlib
       - librosa
       - gitpython
+      - pyyaml
 prefix: /home/znovak/miniconda3/envs/asid
```

Comment:
remove prefix

Reply:
This is the package. `pip install yaml` doesn't work

Reply:
oh i meant the line under it